crates/settlement-clients/ethereum/src/lib.rs

```diff
@@ -198,11 +217,20 @@ impl SettlementClient for EthereumSettlementClient {
 
         let max_fee_per_blob_gas: u128 = self.provider.get_blob_base_fee().await?.to_string().parse()?;
 
+        // calculating y_0 point
+        let y_0_point_u256 = convert_stark_bigint_to_u256(
+            bytes_to_u128(&program_output[Y_LOW_POINT_OFFSET]),
+            bytes_to_u128(&program_output[Y_HIGH_POINT_OFFSET]),
+        );
+        let y_0_point_bytes_32 = Bytes32::from(y_0_point_u256.to_be_bytes());
```

Comment:
you don't need to append type on var name in general, linters show the type

Reply:
you could just use `y_0` at both places (second delcaration wll override the other, which is nice because first one is only used to create the second one)